volume-providers/vpc/provider/util.go

```diff
@@ -86,6 +90,34 @@ func skipRetry(err *models.Error) bool {
 	return false
 }
 
+// skipRetryForIKS skip retry as per listed error codes
+func skipRetryForIKS(err *models.IksError) bool {
```

Comment:
func skipRetryForIKS(err *models.IksError) bool 
func skipRetry

I see these 2 functions...... in skipRetry , the code loops through multiple error codes  and  skipRetryForIKS, i see skip errorcodes being used from a constant Variable. why the behavior is different ?

Reply:
As of now MS(or any carrier ms) is returning only one error thats why